content/v2.0/security/enable-tls.md

```diff
@@ -13,44 +13,61 @@ v2.0/tags: [security, authentication, tls, https, ssl]
 Enabling HTTPS encrypts the communication between clients and the InfluxDB server.
 When configured with a signed certificate, HTTPS can also verify the authenticity of the InfluxDB server to connecting clients.
```

Comment:
When the InfluxDB server is configured with a signed certificate, clients can verify the authenticity of the InfluxDB server?

Reply:
👍